doc/1/controllers/document/replace/index.md

```diff
@@ -0,0 +1,48 @@
+---
+code: true
+type: page
+title: ReplaceAsync
+description: Replaces a document.
+---
+
+# ReplaceAsync
+
+Replaces the content of an existing document.
+
+## Arguments
+
+```csharp
+public async Task<JObject> ReplaceAsync(
+  string index, string collection, string id, JObject content, bool waitForRefresh = false );
+
+```
+
+<br/>
+
+| Argument     | Type                                 | Description                           |
+| ------------ | ------------------------------------ | ------------------------------------- |
+| `index`      | <pre>string</pre>        | Index name                            |
+| `collection` | <pre>string</pre>        | Collection name                       |
+| `id`         | <pre>string</pre>        | Document ID                           |
+| `document`   | <pre>JObject</pre>        | JObject representing the document |
+| `waitForRefresh`   | <pre>bool</pre><br/>(`false`)       | If `true`, waits for the change to be reflected for `search` (up to 1s)           |
```

Comment:
(nitpicking) I prefer the newest description for that option, as seen in the API documentation:

```suggestion
| `waitForRefresh`   | <pre>bool</pre><br/>(`false`)       | if `true`, Kuzzle will not respond until the newly replaced document is indexed  |
```

If you approve of that change, then obviously this must be propagated to all `waitForRefresh` descriptions.